### ISSUE TITLE
Alt+Space issues with Ulauncher

### ISSUE THREAD

**Issue by RedBearAK** [rbreaves/kinto#228]
Twister OS (32-bit Raspbian/Debian)
XFCE
Ulauncher

Trying to assign physical Alt+Space to activate Ulauncher so I can have that "Spotlight-like" experience. Similar to the issue with Alt+Grave, the key combo doesn't work when Kinto is active. Works fine when Kinto is disabled. When Kinto is enabled, can't even assign the key combo to Ulauncher's capture mechanism. 

Tried setting the key combo directly in Ulauncher's settings.json file, replacing: 

`"hotkey-show-app": "<Alt>space",`

with

`"hotkey-show-app": "<Ctrl>space",`

No dice. 

But, again just like the Alt+Grave issue, I can leave it at Alt+Space and it works (with physical Win+Space). But it seems Ctrl+Space simply will not work when Kinto is active. 

Perhaps there is another magic key that will make this work? I don't even understand why this is happening since there is no reference at all to the Space key in kinto.py, only references to Backspace and Grave. Yet this seems a very similar issue. Somehow this key combination (physical Alt+Space, virtual Ctrl+Space) seems to be unable to be used, or even seen, by Ulauncher. 

Kinto isn't changing the space key in any way, right? The same key code is shown for the space key by xbindkeys whether Kinto is enabled or disabled. 

The only real difference I can see between Kinto enabled and disabled is this: 

`    m:0x4 + c:37
    Control + Control_L
`
This is with Kinto disabled. 

`    m:0x4 + c:105
    Control + Control_R
`
This is with Kinto enabled. Is it a necessity to map Alt to Control_R rather than Control_L? Could that possibly make any difference to any piece of software? 

I thought maybe a default XFCE keyboard shortcut could be interfering, but can't find any Space-related keyboard shortcut in Window Manager. 

ADDITIONAL INFO: Discovered after assigning (phys) Alt+Shift+U that Ulauncher saves this in its settings file as `"<Primary><Shift>u"`. Progress? I have no idea! 

Trying to change this to `"<Primary>space"` results in an error message of `"Binding '<Primary>space' failed!"` when launching Ulauncher via terminal. `"<Primary><Shift>space"` also produces a similar error message. `"<Primary><Shift>Space"` or` "<Primary><Shift><Space>"` produce no error messages but also fail to function. At least one of these left Ulauncher activating when I simply pressed the Alt key. Or was it the space bar? 

So something is causing the `"<Primary>space"` or `"<Primary><Shift>space"` binding to fail. Maybe this doesn't even have anything to do with Kinto? I also have no idea! 

`"<Primary><Alt>space"` works! So close, so close, and yet so far. 

I'll keep trying.

**Comment by RedBearAK** [comment]
Argh. Might just be Ulauncher at the root of this problem. Found this in the Ulauncher github: 

**gornostal commented on Mar 10, 2017
Sorry Alt+Space won't work with Ulauncher.

Cerebro app uses totally different tools for key bindings + the most popular Linux DEs already use Alt+Space to show a window menu.**

**Comment by RedBearAK** [comment]
I just figured something out. Should have taken a screenshot, because all I can remember is something about "whisker menu". Anyway, in XFCE most of the system keyboard shortcuts are in Window Manager -> Keyboard. But there are also additional shortcuts in Keyboard -> Application Shortcuts. A couple of them were assigned to Ctrl+Space and Shift+Ctrl+Space. Also there was something about activating the menu which was set to "Super L". Once I deleted all of these, I can now assign Alt+Space to Ulauncher. No problem. 

Kinto (probably) had nothing whatsoever to do with it.
